clash-prelude/tests/Clash/Tests/Signal.hs

```diff
@@ -216,8 +217,14 @@ case_dynamicStaticEq = do
 
     clk11 = clockGen @H11
     clk77 = clockGen @H77
-    dclk11 = dynamicClockGen @H11 (pure (hzToPeriod 11))
-    dclk77 = dynamicClockGen @H77 (pure (hzToPeriod 77))
+
+    -- We construct periods in a roundabout way (i.e., using 'hzToPeriod' instead
+    -- of using 'hzToPs'), to prevent rounding errors.
```

Comment:
```suggestion
    -- of using 'hzToPs'), to prevent rounding errors between periods of the static clocks and the initial periods of the dynamic clocks.
```

Reply:
Not only the initial periods, but broader point taken!